watertap/unit_models/zero_order/nanofiltration_zo.py

```diff
@@ -0,0 +1,73 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module constians a ero-order representation of a nanofiltration unit
+operation.
+"""
+
+from pyomo.environ import Constraint, units as pyunits, Var
+from idaes.core import declare_process_block_class
+
+from watertap.core.zero_order_sito import SITOBaseData
+
+# Some more inforation about this module
+__author__ = "Andrew Lee"
+
+
+@declare_process_block_class("NanofiltrationZO")
+class NanofiltrationZOData(SITOBaseData):
+    """
+    Zero-Order model for a Nanofiltration unit operation.
+    """
+
+    CONFIG = SITOBaseData.CONFIG()
+
+    def build(self):
+        self._has_deltaP_treated = True
+        self._has_deltaP_byproduct = True
+
+        super().build()
+
+        # Add electricity consumption to model
+        self.electricity = Var(self.flowsheet().time,
+                               units=pyunits.kW/pyunits.hour,
```

Comment:
This _can_ be in kW to represent _power_ consumption or kWh to represent energy consumption. If doing the latter, you could just multiply by hour in the constraint. However, kW/hour should not be used.